docs/Ivan_CLI.md

```diff
@@ -0,0 +1,240 @@
+## Command Line Interface (CLI)
+
+Walking side by side with the Python client, the Inductiva CLI provides 
+a quick and simple interface to interact and perform the most common tasks from
+the terminal. The CLI allows users to quickly manage their tasks, resources and/or
+storage, while their simulations are still running remotely.
+
+### Overview of the CLI
+
+The CLI beauty is that its functionalities are a terminal command distance away.
+By doing,
+```bash
+inductiva
+```
+
+you will obtain an overview of the available commands and their descriptions.
+
+Here, we highlight and go deeper on the four main functionalities of the CLI:
+- [`resources`](#resource-management-in-the-cli) gives management capabilities over the resources;
+- [`tasks`](#task-management) empowers the quick management of the simulations;
+- [`logs`](#streaming-logs-of-a-task) allows users to stream the logs of their running tasks;
+- [`storage`](#remote-storage-overview) provides an overview of the user's remote storage.
+
+
+Whenever you need help on a specific command, you can always use the `--help` or
+`-h` flag to further understand its usage.
+
+#### Resource Management in the CLI
+
+As you might know by now, Inductiva API provides a simple way to [launch
+dedicated resources]() where you can run your simulations. Before launching any
+resources, users can use the CLI to gather more information about the right
+resources to launch with the `available` and `cost` subcommands.
+
+With them, user can list all the available machine types together with details,
+```bash
+$ inductiva resources available
+Available machine types
+
+machine-type: [cores-available]
```

Comment:
We need to fix this :)
I want to see here the c3 machines (even if they blow up user quota!)

Reply:
I'm on it!